specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/scriptActions.json

```diff
@@ -0,0 +1,420 @@
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK response definition.",
+                        "schema": {
+                            "$ref": "#/definitions/ScriptActionsList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/scriptExecutionHistory/{scriptExecutionId}": {
+            "get": {
+                "tags": [
+                    "ScriptExecutionHistory"
+                ],
+                "operationId": "ScriptActions_GetExecutionDetail",
+                "description": "Gets the script execution detail for the given script execution ID.",
+                "x-ms-examples": {
+                    "Get script execution history by script id": {
+                        "$ref": "./examples/GetScriptActionById.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ClusterNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ScriptExecutionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK response definition.",
+                        "schema": {
+                            "$ref": "./cluster.json#/definitions/RuntimeScriptActionDetail"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/scriptExecutionHistory": {
+            "get": {
+                "tags": [
+                    "ScriptExecutionHistory"
+                ],
+                "operationId": "ScriptExecutionHistory_List",

```

Comment:
>ScriptExecutionHistory_List [](start = 32, length = 27)

Operation name should be "ScriptExecutionHistory_ListByCluster"

Reply:
Done.